### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by SebastianRuffert**

### CODE REVIEW

**Comment by fhagemann** [comment]
Nice changes!

**Comment by SebastianRuffert** [comment]
Thanks!

File: docs/src/tutorial_lit.jl [hunk 1]
```diff
@@ -116,7 +116,7 @@ plot_electric_fieldlines!(sim, full_det = true, φ = 0.0)
 charge_drift_model = ADLChargeDriftModel()
 sim.detector = SolidStateDetector(sim.detector, charge_drift_model)
 
-# Now, let's create an "random" multi-site event:
+# Now, let's create a "random" multi-site event:
```
Comment (fhagemann) [inline docs/src/tutorial_lit.jl:119]: Thanks for correcting!

File: examples/example_config_files/test_geometry.yaml [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+      from: -10
+      to: 10
+      boundaries: reflecting
+medium: vacuum
+detectors:
+- semiconductor:
+    material: HPGe
+    temperature: 78
+    impurity_density:
+      name: linear
```
Comment (SebastianRuffert) [inline examples/example_config_files/test_geometry.yaml:28]: should we user a linear impurity density here?

**Review by fhagemann — APPROVED** [review]

**Pull request closed without merging**